barc_blanket/models/simple_geometry.py

```diff
@@ -1,2 +1,187 @@
+blanket1_cell.region = +blanket_tank & -tank_divider_in
+blanket1_cell.fill = flibe
+
+tank_divider_cell = openmc.Cell()
+tank_divider_cell.region = +tank_divider_in & -tank_divider_out
+tank_divider_cell.fill = inconel718
+
+blanket2_cell = openmc.Cell()
+blanket2_cell.region = +tank_divider_out & -burner_tank
+blanket2_cell.fill = flibe
+
+container_cell = openmc.Cell()
+container_cell.region = +burner_tank & -bounding_sphere
+
+universe = openmc.Universe()
+universe.add_cell(plasma_cell)
+universe.add_cell(sol_cell)
+universe.add_cell(vv_cell)
+universe.add_cell(blanket1_cell)
+universe.add_cell(tank_divider_cell)
+universe.add_cell(blanket2_cell)
+universe.add_cell(container_cell)
+
+# %% Source Definition
+
+source = openmc.IndependentSource()
+source.particle = 'neutron'
+radius = openmc.stats.Discrete([680], [1]) # centered at major radius
+z_values = openmc.stats.Discrete([0], [1])
+angle = openmc.stats.Uniform(a=np.radians(0), b=np.radians(360))
+source.space = openmc.stats.CylindricalIndependent(
+    r=radius, phi=angle, z=z_values, origin=(0., 0., 0.))
+source.angle = openmc.stats.Isotropic()
+source.energy = openmc.stats.muir(e0=14.08e6, m_rat=5, kt=20000)
+
+# %% Settings
+
+settings = openmc.Settings(run_mode='fixed source')
+settings.photon_transport = False
+settings.source = source
+settings.batches = 50
+settings.particles = int(1e5) # modify this to shorten simulation, default was 1e6 
+settings.statepoint = {'batches': [
+    5, 10, 15, 20, 25, 30, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90, 95, 100]}
+settings.output = {'tallies': True}
+
+# %% Filters and Tallies
+
+burner_cell_filter = openmc.CellFilter(blanket2_cell) # just burner blanket
+tbr_cell_filter = openmc.CellFilter([blanket1_cell,blanket2_cell]) # includes both fusion and burner
+energy_filter = openmc.EnergyFilter(np.logspace(0,7)) # 1eV to 100MeV
+
+# mesh tally - flux
+
+tally1 = openmc.Tally(tally_id=1, name="flux_burner")
+tally1.filters = [burner_cell_filter,energy_filter]
+tally1.scores = ["flux"]
+
+# tbr
+
+tally2 = openmc.Tally(tally_id=2, name="tbr")
+tally2.filters = [tbr_cell_filter]
+tally2.scores = ["(n,Xt)"]
+
+#power deposition - heating-local
+
+tally3 = openmc.Tally(tally_id=3, name="heating_burner")
+tally3.filters = [burner_cell_filter]
+tally3.scores = ["heating-local"]
+# %% 
+
+# Define materials, geomery, and tallies
+materials = openmc.Materials([dt_plasma,flibe,inconel718,eurofer])
+geometry = openmc.Geometry(universe)
+tallies = openmc.Tallies([tally1,tally2,tally3])    
+
+# Download cross cection data for materials
+materials.download_cross_section_data(
+        libraries=['ENDFB-7.1-NNDC', 'TENDL-2019'],
+        set_OPENMC_CROSS_SECTIONS=True,
+        particles=["neutron"],
+    )
+
+model = openmc.Model(materials=materials, geometry=geometry,
+                     settings=settings, tallies=tallies)
+
+# Export and run model
+model.export_to_model_xml()
+model.run(threads=8, geometry_debug=True)
```

Comment:
This will allow you to import this script without running the model.
Whatever is in `if __name__=="__main__":` will only be run if you run this specific script. Allowing you to import objects in the notebook without running this long step.

```suggestion
# Export and run model
if __name__=="__main__":
    model.export_to_model_xml()
    model.run(threads=8, geometry_debug=True)
```